runtime/jailer.go

```diff
@@ -0,0 +1,299 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"syscall"
+
+	"github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"golang.org/x/sys/unix"
+
+	"github.com/firecracker-microvm/firecracker-go-sdk"
+)
+
+const (
+	jailerFolder        = "jail"
+	firecrackerBinName  = "firecracker"
+	kernelImageFileName = "kernel-image"
+	jailerHandlerName   = "firecracker-containerd-jail-handler"
+	runcConfigPath      = "/etc/containerd/firecracker-runc-config.json"
```

Comment:
Is this something that can be overridden by users? Doesn't have to be in the PR but we should definitely let them optionally use a different path